tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
+    "module": "commonjs",
+    "target": "es6",
```

Comment:
@Wykks Yep, totally correct, except TS does compile ES6 to ES5, except for very specific staged things like: Iterables / Iterators, Async / Await & Generators